src/_gettsim/parameters/erwerbsm_rente.yaml

```
@@ -0,0 +1,256 @@
+---
+altersgrenze_abschlagsfrei:
+  name:
+    de: Regelaltersgrenze
+    en: Statutory retirement age
+  description:
+    de: >-
+      Altersgrenze bis zu der man die Erwerbsminderungsrente ohne Abzüge in
+      Anspruch nehmen kann. Bis 2012 lag die Regelaltersgrenze bei 63. Ab dann
+      schrittweise Anhebung bis 65.
+    en: >-
+      Age limit up to which one can claim the public disability insurance without
+      deductions. Up to 2012, the standard retirement age was 63.
+      From then on gradual increase until 65.
+  unit: Year
+  reference: § 77 SGB VI Zugangsfaktor
+  note: >-
+    "https://sozialversicherung-kompetent.de/rentenversicherung/leistungsrecht/
+    295-erwerbsminderungsrente-und-rentenabschlaege.html"
+  2001-01-01:
+    scalar: 63
+  2012-01-01:
+    scalar: 63.083333
+  2012-02-01:
+    scalar: 63.166666
+  2012-03-01:
+    scalar: 63.25
+  2012-04-01:
+    scalar: 63.333333
+  2012-05-01:
+    scalar: 63.416666
+  2012-06-01:
+    scalar: 63.5
+  2013-01-01:
+    scalar: 63.583333
+  2014-01-01:
+    scalar: 63.666666
+  2015-01-01:
+    scalar: 63.75
+  2016-01-01:
+    scalar: 63.833333
+  2017-01-01:
+    scalar: 63.916666
+  2018-01-01:
+    scalar: 64
+  2019-01-01:
+    scalar: 64.166666
+  2020-01-01:
+    scalar: 64.333333
+  2021-01-01:
+    scalar: 64.5
+  2022-01-01:
+    scalar: 64.666666
+  2023-01-01:
+    scalar: 64.833333
+  2024-01-01:
+    scalar: 65
+altersgrenze_langj_versicherte_abschlagsfrei:
+  name:
+    de: Regelaltersgrenze für langjährige Versicherte
+    en: Statutory retirement age for long-term insured persons
+  description:
+    de: >-
+      Altersgrenze bis zu der man die Erwerbsminderungsrente ohne Abzüge in
+      Anspruch nehmen kann wenn man bestimmte zeitliche Bedingungen
+      erfüllt. Bedingungen werden in § 77 Abs. 4 SGB VI definiert.
+    en: >-
+      Age limit up to which one can claim the public disability insurance
+      without deductions under certain time conditions.
+      These conditions are defined in § 77 4 SGB VI.
+  unit: Year
+  reference: § 77 Abs. 4 SGB VI Zugangsfaktor
+  2001-01-01:
+    scalar: 63
+zurechnungszeitsgrenze:
```

```suggestion
zurechnungszeitgrenze:
```

Please watch spelling a bit more closely. This really is unnecessary work for reviewers.